api/license.yaml

```diff
@@ -4,6 +4,17 @@ group:
   name: license
   description: OEE license management
 commands:
+  - name: info
+    description: Get license info
+    descriptionLong: Get license status and expiration date
+    method: GET
+    path: service/apama-oeeapp/mon/info/license
```

Comment:
didnt work for me; got the error message:
```
PS C:\Users\mvoigt>  c8y oee-cli.naph license info
2023-05-25T11:05:53.257+0200    ERROR   serverError: GET https://mobile-mvoigt.eu-latest.cumulocity.com/service/apama-oeeapp/mon/info/license?pageSize=20: 404
```
I think the trailing `?pageSize=20` is causing this problem...